heudiconv/dicoms.py

```diff
@@ -364,11 +401,11 @@ def compress_dicoms(dicom_list, out_prefix, tempdirs, overwrite):
     # Solution from DataLad although ugly enough:
 
     dicom_list = sorted(dicom_list)
-    dcm_time = get_dicom_series_time(dicom_list)
+    dcm_time = get_timestamp_from_series(dicom_list)
 
-    def _assign_dicom_time(ti):
-        # Reset the date to match the one from the dicom, not from the
-        # filesystem so we could sort reproducibly
+    def _assign_dicom_time(ti: tarfile.TarInfo) -> tarfile.TarInfo:
+        # Try to reset the date to match the one of the last commit, not from the
```

Comment:
what commit? there is no assumption in heudiconv on having dataset in git here. And is this TODO?

Reply:
That's a comment that was there when I forked initially. I see that since then [you've updated the note](https://github.com/nipy/heudiconv/commit/5db2ede88280635dd39f58d283e5bb1fb480861f). When updating this pull request I'll make sure to integrate the more recent commits/handle any merge conflicts